database/migrations/000042_credential_enrollment_nonce.up.sql

```diff
@@ -0,0 +1,28 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+ALTER TABLE provider_github_app_installations ADD COLUMN enrollment_nonce TEXT;
+
+ALTER TABLE provider_github_app_installations ADD COLUMN project_id UUID;
+
+ALTER TABLE provider_github_app_installations
+    ADD CONSTRAINT provider_github_app_installations_project_id_fkey
+        FOREIGN KEY (project_id)
+            REFERENCES projects(id);
```

Comment:
Do we want `ON DELETE CASCADE`?

Reply:
I also thought about this. We have an `ON DELETE CASCADE` on the provider, which implicitly means that if a project is deleted, then the provider is deleted, then the installation is deleted. I'm not sure if it's better to make it explicit here.